src/plugins/intel_cpu/src/transformations/snippets/x64/op/brgemm_utils.hpp

```diff
@@ -42,14 +43,19 @@ size_t compute_vnni_factor(const ov::element::Type& precision);
 size_t get_elems_in_vec(const ov::element::Type& precision);
 
 namespace repacking {
+/// \brief  Computes inner N block size used by OneDNN implementation. Depends on tensor precision
+size_t compute_inner_n_block(const ov::element::Type& precision);
```

Comment:
Minor: What's about to rename this function to `compute_wei_n_blk` (oneDNN uses `wei_n_blk` to describe this block size) to align with the onednn structures for easier debugging and comparison?

Reply:
And the same for `compute_out_leading_dim`: can we rename to `compute_LDB`? At the moment we have only `BrgemmCopyB` with `LDB` (stride on second input of `Brgemm`). But in the near future, we will implement `BrgemmCopyA` with `LDA` (stride on first input of `Brgemm`) and we will have the similar two methods `compute_out_leading_dim` but for different repacking ops.

Reply:
I think it's a good idea to rename `compute_out_leading_dim`, since we use it for inputs as well.
Not sure about `wei_n_blk`, since `compute_inner_n_block` is more explicit. It distinguishes between N block size, and inner block size.